x11-packages/qt5-qtwebengine/build.sh

```diff
@@ -0,0 +1,77 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/qt/qtwebengine
+TERMUX_PKG_DESCRIPTION="Qt 5 Web Engine Library"
+TERMUX_PKG_LICENSE="LGPL-3.0, LGPL-2.1, BSD 3-Clause"
+TERMUX_PKG_MAINTAINER="@termux"
```

Comment:
Who maintain(s) this elephantine package?

Reply:
Should I set it to myself?

Reply:
Set maintainer to me now.